GPU/GPUCommon.cpp

```diff
@@ -570,6 +570,8 @@ void GPUCommon::DumpNextFrame() {
 }
 
 u32 GPUCommon::DrawSync(int mode) {
+	drawEngineCommon_->DrawSync();
```

Comment:
Should GLES do this too?

Personally, I'd prefer to clear `prevDcid_` each time we start a new interpret cycle, in case games modify vert data between.  Do the games you're targeting reuse between or only within a single stall?

-[Unknown]

Reply:
Yeah, maybe that's a better time to clear prevDcid_. I'm still experimenting.